kratos/processes/simple_mortar_mapper_process.h

```diff
@@ -675,6 +676,9 @@ class KRATOS_API(KRATOS_CORE) SimpleMortarMapperProcess
         // Indexes of the pair to be removed
         std::vector<IndexType> indexes_to_remove, conditions_to_erase;
 
+        // Getting discontinous factor
+        const double discontinous_interface_factor = mDiscontinuousInterface ? mThisParameters["discontinous_interface_factor"].GetDouble() : 1.0;
+
         // Geometrical values
         const array_1d<double, 3>& r_slave_normal = itCond->GetValue(NORMAL);
         GeometryType& r_slave_geometry = itCond->GetGeometry();
```

Comment:
```suggestion
        const auto &r_slave_geometry = itCond->GetGeometry();
```
Not sure at all about the `const` (I went through the code but didn't try).

Reply:
I cannot add the const, intermediate utilities are not const, that can be changed on a future PR